src/extensions/default/InlineColorEditor/main.js

```diff
@@ -24,11 +24,34 @@
 define(function (require, exports, module) {
     "use strict";
 
-    var EditorManager       = brackets.getModule("editor/EditorManager"),
-        ExtensionUtils      = brackets.getModule("utils/ExtensionUtils"),
-        InlineColorEditor   = require("InlineColorEditor").InlineColorEditor,
-        ColorUtils          = brackets.getModule("utils/ColorUtils");
+    try {
+        var EditorManager       = brackets.getModule("editor/EditorManager"),
+            ExtensionUtils      = brackets.getModule("utils/ExtensionUtils"),
+            InlineColorEditor   = require("InlineColorEditor").InlineColorEditor,
+            ColorUtils          = brackets.getModule("utils/ColorUtils"),
+            ColorProperties     = require("text!ColorProperties.json"),
+            properties          = JSON.parse(ColorProperties);        
```

Comment:
Let's not but all this in the `try/catch`.  The only thing that might fail is `JSON.parse()`, but even that I'm not too worried about--since this file doesn't change, and we'll catch it in debug builds.  Also, Brackets uses this pattern all over the place:

```js
var properties = JSON.parse(require("text!ColorProperties.json"));
```

So let's just get rid of the `try/catch` block completely.